decaylanguage/data/decfile.lark

```diff
@@ -3,7 +3,7 @@
 // Distributed under the 3-clause BSD license, see accompanying file LICENSE
 // or https://github.com/scikit-hep/decaylanguage for details.
 
-start : _NEWLINE? (line _NEWLINE)+ ("End" _NEWLINE)?
+start : _NEWLINE* (line _NEWLINE+)+ ("End" _NEWLINE+)?
```

Comment:
Why a `+` in the `_NEWLINE+` after `End`?

Reply:
That would allow End followed by multiple newlines, but what is after End doesn't matter anyway IIRC, so I think it could safely go either way.

Reply:
Indeed, since empty lines are ignored. If anything, without the + the parsing will be an epsilon faster.